server/src/QRServer/config.py

```diff
@@ -75,6 +75,13 @@ def get_type(self):
     default_value=False,
     requires_restart=False)
 
+discord_webhook_lobby_url = ConfigKey(
```

Comment:
I'd adopt slightly different approach that would make it possible to define custom messages, webhook name and icon for each message type. Basic use-case for custom formatting would be to add role pings. I'd suggest config class with default messages and option to add custom ones and then do something like 
```message.format(<list of parameters>)```, which would make it possible to use any of the provided params in message.

Reply:
See [this](https://discord.com/developers/docs/resources/webhook#execute-webhook-jsonform-params) for possible params, I'd add content, username and avatar_url as customizable options

Reply:
I think it's a good start, we could merge it and then improve upon it

Reply:
fair enough